modules/integration_aws-backup/conf/02-aws-backup-copyjob-failed.yaml

```diff
@@ -0,0 +1,22 @@
+module: AWS Backup
```

Comment:
most of the comments from the previous detector apply here too.

Reply:
Addressed: https://github.com/claranet/terraform-signalfx-detectors/pull/457/files#diff-db6650db0c70872fdc7594d15948ddd6fa7a26897c2bd0c2f5ef887cf12008b1R5-R16